File: 기초-브론즈 문제/백준_3052_나머지/나머지.cpp
#include <iostream>

using namespace std;

int main() {
    int arr[42] = {0};
    int n = 10;
    int a = 0;
    while(n--){
        scanf("%d", &a);
        arr[a%42]++;
    }
    int cnt = 0;
    for(int i = 0; i< 42; i++){
        if(arr[i] > 0) cnt++;
    }
    printf("%d", cnt);
}

File: 기초-브론즈 문제/백준_3052_나머지/나머지.py
arr = [0 for _ in range (42)]
a = 0;
num = 10

while num > 0:
    num -=1
    a = int(input())
    arr[a%42] += 1

cnt = 0
for i in range(42):
    if arr[i] > 0: cnt += 1

print(cnt)